src/main/java/teammates/storage/api/EntitiesDb.java

```diff
@@ -141,6 +141,52 @@ public A putEntity(A entityToAdd) throws InvalidParametersException {
         return makeAttributes(entities);
     }
 
+    /**
+     * Creates a collection of entities in the datastore without existence checking.
+     * The call is silent about invalid/existing entities.
+     *
+     * <p>The documents of the associated entities (if applicable) WILL NOT be updated.
+     *
+     * @return created entities
+     */
+    public List<A> createEntitiesSilent(Collection<A> entitiesToAdd) {
+        return createEntitiesSilent(entitiesToAdd, true);
+    }
+
+    /**
+     * Creates a collection of entities in the datastore without existence checking.
```

Comment:
I'm quite confused here since the comments say `without existence checking`, but `shouldCheckExistence` is set to true above

Reply:
Ah good catch; when porting over the changes I made from the previous PR, I believe there was some intermediary changes that weren't captured. Thanks!